crawler/plugins/disk_crawler.py

```diff
@@ -9,13 +9,16 @@
 def crawl_disk_partitions():
     partitions = []
     for partition in psutil.disk_partitions(all=True):
-        pdiskusage = psutil.disk_usage(partition.mountpoint)
-        partitions.append((partition.mountpoint, DiskFeature(
-            partition.device,
-            100.0 - pdiskusage.percent,
-            partition.fstype,
-            partition.mountpoint,
-            partition.opts,
-            pdiskusage.total,
-        ), 'disk'))
+        try:
+            pdiskusage = psutil.disk_usage(partition.mountpoint)
+            partitions.append((partition.mountpoint, DiskFeature(
+                partition.device,
+                100.0 - pdiskusage.percent,
+                partition.fstype,
+                partition.mountpoint,
+                partition.opts,
+                pdiskusage.total,
+            ), 'disk'))
+        except OSError:
```

Comment:
If one of the partitions in psutil.disk_partitions disappears while we are in the loop, an OSError is triggered.